diff --git a/webapp/node_modules/ts-loader/index.js b/webapp/node_modules/ts-loader/index.js
index 5f52239..08b1a88 100644
--- a/webapp/node_modules/ts-loader/index.js
+++ b/webapp/node_modules/ts-loader/index.js
@@ -193,8 +193,10 @@ function loader(contents) {
     if (!file) {
         file = instance.files[filePath] = { version: 0 };
     }
-    file.text = contents;
-    file.version++;
+    if (file.text !== contents) {
+        file.text = contents;
+        file.version++;
+    }
     // Make this file dependent on *all* definition files in the program
     this.clearDependencies();
     this.addDependency(filePath);
